### ISSUE TITLE
Run training on the three years dataset

### ISSUE THREAD

**Issue by rogerkuou** [ESMValGroup/ClimaNet#25]
Data at `/work/bd0854/b380103/eso4clima/output/concatenated`

- [x] Adapat the current example notebook to run with the two year data
- [x] #28 (made a new issue)

Deliver
- [x] Adapted notebook on a fraction of the two year data
- [x] Python example to run on the full two year data
- [x] A SLURM example to run the Python script on Levante

**Comment by SarahAlidoost** [comment]
For training on three years, both memory and runtime should be handled:

- use pre-processed data at `/work/<account_id>/eso4clima/preprocessed/sst/"` and combine three years of data as: `xr.concat([da_2018, da_2019, da_2020], dim="M")`, keep 2021 for validation and 2022 for test.
- in slurm script, use `--mem 0` , and `--constraint=a100_80`, see [this example](https://github.com/ESMValGroup/ClimaNet/blob/main/scripts/tuning.slurm).
- in dataset, use `load_lazy=True` and in dataloader:

```python
dataloader = DataLoader(
    dataset_train,
    batch_size=500,  # or more
    shuffle=True,
    pin_memory=True,  # when on gpu
    num_workers=32,  # for data loading
    persistent_workers=True,  # keep workers alive between epochs
    multiprocessing_context="spawn", # keep this when num_workers >0 and lazy data
    
)
```

Note:
- set all verbose arguments to False to have a nicer log file.

**Comment by rogerkuou** [comment]
Hi @SarahAlidoost , FYI I could not access `/work/<account_id>/eso4clima/preprocessed/sst/`, got a permission denied error.

I used the data_preparation work flow and duplicated a 5 year preprocessed data under my accout. 

I also updated the traing script in  https://github.com/ESMValGroup/ClimaNet/pull/27 to the best state I can. A training job is now queueing on Levante.

**Comment by SarahAlidoost** [comment]
> Hi [@SarahAlidoost](https://github.com/SarahAlidoost) , FYI I could not access `/work/<account_id>/eso4clima/preprocessed/sst/`, got a permission denied error.

@rogerkuou Hi, the directory has permission `drwxr-xr-x` meaning group members can copy it to scratch dir. Did you replace `<account_id>` with our Levante project id?

**Comment by SarahAlidoost** [comment]
> I also updated the traing script in [#27](https://github.com/ESMValGroup/ClimaNet/pull/27) to the best state I can. A training job is now queueing on Levante.

I dont see the settings that I explained [here](https://github.com/ESMValGroup/ClimaNet/issues/25#issuecomment-5241361005) in PR #27. Can you please check?

**Comment by rogerkuou** [comment]
> > Hi [@SarahAlidoost](https://github.com/SarahAlidoost) , FYI I could not access `/work/<account_id>/eso4clima/preprocessed/sst/`, got a permission denied error.
> 
> [@rogerkuou](https://github.com/rogerkuou) Hi, the directory has permission `drwxr-xr-x` meaning group members can copy it to scratch dir. Did you replace `<account_id>` with our Levante project id?

Thanks @SarahAlidoost ! I just tried again, it works now. However the same path was not accessible on past Sunday. I suspect it relates to this issue with Infiniband and Storage (received Levante email yesterday). But it works now.

**Comment by rogerkuou** [comment]
@SarahAlidoost One job I submitted this morning failed with OOM error:

I put the logs and scripts in this path:

`/work/<account_id>/eso4clima/train_three_years/experiment_20260818`

Given I wont have availability till Tursday, would you like to pick it up?
